laboratory-servlet-api-integration/src/main/scala/com/wixpress/petri/laboratory/AmplitudeTestGroupAssignmentTracker.scala

```diff
@@ -21,10 +22,10 @@ class AmplitudeTestGroupAssignmentTracker(amplitudeAdapter: AmplitudeAdapter) ex
 case class AmplitudeAdapter(amplitudeUrl: String, apiKey: String, amplitudeTimeoutMs: Int) {
   def this(amplitudeUrl: String, apiKey: String) = this(amplitudeUrl, apiKey, 2000)
 
-  val requestConfig = RequestConfig.custom.setConnectTimeout(amplitudeTimeoutMs).build
-  val client = HttpClientBuilder.create.setDefaultRequestConfig(requestConfig).build
+  val requestConfig = RequestConfig.custom.setConnectionRequestTimeout(amplitudeTimeoutMs).build
```

Comment:
interesting. what happened to make you change this?

and shouldnt it still also set the connectTimeout?